test/ClangImporter/Inputs/restated-protocol-requirement.h

```diff
@@ -0,0 +1,8 @@
+@protocol P
+@property (nonatomic, copy, readonly, nullable) id name;
+@end
+
+@protocol Q <P>
+@property (nonatomic, copy, readonly, nullable) id name;
```

Comment:
Do we have a similar problem for methods or subscripts or initializers?

Reply:
I've added tests.